api/README.md

```diff
@@ -1283,3 +1286,75 @@ Content-Type: application/json
   { "id": "id3", "doc": { <...the hydrated document...> } },
 ]
 ```
+
+
+# Contacts by phone
+
+Accepts a phone number parameter and returns fully hydrated contacts that match the requested phone number.
+If multiple contacts are found, all are returned.  When no matches are found, a 404 error is returned.
```

Comment:
We'll have multiple contacts per phone# when this hits: https://github.com/medic/cht-core/issues/6390